pkg/kube-metrics/store.go

```diff
@@ -23,26 +23,24 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/tools/cache"
-	kcollector "k8s.io/kube-state-metrics/pkg/collector"
 	"k8s.io/kube-state-metrics/pkg/metric"
 	metricsstore "k8s.io/kube-state-metrics/pkg/metrics_store"
 )
 
-// NewCollectors returns collections of metrics in the namespaces provided, per the api/kind resource.
+// NewMetricsStores returns collections of metrics in the namespaces provided, per the api/kind resource.
 // The metrics are registered in the custom generateStore function that needs to be defined.
```

Comment:
Nit:
```suggestion
// The metrics are registered in the custom metrics.FamilyGenerator that needs to be defined.
```
I think this is outdated from the time of the original proposal. There is no generateStore func passed in.
It's the metrics.FamilyGenerator that should be defined for that kind.
https://github.com/operator-framework/operator-sdk/blob/master/pkg/kube-metrics/metrics.go#L69